src/applications/personalization/preferences/containers/SetPreferences.jsx

```diff
@@ -71,6 +72,15 @@ class SetPreferences extends React.Component {
     this.props.setPreference(code, !this.props.preferences.dashboard[code]);
   };
 
+  // checks to see if the current state of the dashboard (ie preferences
+  // selected by the user) is different from how they were when they were pulled
+  // from the server
+  userHasNotMadeChange = () =>
+    isEqual(
+      this.props.preferences.dashboard,
+      this.props.preferences.dashboardBackup,
```

Comment:
Should this be `dashboardArchive` or `savedDashboard`, backup doesn't quite communicate what is being backed up? We can address this later if we can come up with something that seems more descriptive. perhaps this could be `savedBenefits` and `stagedBenefits` since there will be other preferences in the future, and as dashboard is actually a little generic?

Reply:
Those aren't great names, either. Bottom line is I want to get the naming in better order and more consistent :)